tests/tinypages/.gitignore

```diff
@@ -0,0 +1 @@
+_build/
```

Comment:
Does the example project have to be a git repo? If it doesn't/it isn't, does it need a `.gitignore`?

And what about `tinypages/_static` with an empty `.gitignore`?

Reply:
I'm removing this.  I quite literally copied and pasted this out of matplotlib.